doc/xsd/tester_defs.xsd

```diff
@@ -369,7 +369,7 @@
                 </element>
```

Comment:
summary and description are misleading.
engine/tester: support default args for scripts and sessions

Support run item templates which allow to specify default arguments for
scripts and sessions. These default arguments may be overridden by on
invocation if argument which the same name is specified in run item
arguments.

Moreover script with all its specification may be set in run item template.